### DISCUSSION TITLE
Always show instruction for the next line

### DISCUSSION THREAD

**Discussion by vireakpanha-khy** [outline/rich-markdown-editor#358] [Ideas]
Currently, user have to click on the next line to see the instruction. ```Type \ to insert...```
From my perspective, I think it would be better to have an option that we can always show that instruction in the next line without needing to click on it because sometime, user might confuse and didn't know how to continue writing especially when they write their code and there's no hint to show them that they can continue by just clicking on the next line. If the instruction is there, that will tell the users that they can click on it and continue writing.

**Comment by tommoor** [comment]
I think the question you find yourself asking next is "what is the next line?"

**Reply by vireakpanha-khy** [reply]
I just want to show the instruction in the next line just in case user didn't know how to continue. It's optional anyway.

**Reply by vireakpanha-khy** [reply]
@tommoor By the way, can you show me how to customize tooltip parent div? Is it possible to do that? I've read the document and it seems to be possible but I can't implement it. Can you show me how? (*in ReactJS if possible)

**Reply by tommoor** [reply]
You can see an example of usage in the Outline codebase, 

https://github.com/outline/outline/blob/develop/app/components/Editor.js#L150

**Reply by vireakpanha-khy** [reply]
@tommoor I've read that already. The main problem I want to solve is that user can only change the parent div background color. What if they want to add shadow or border, something like that... or even change the parent style entirely.
